music-journal/components/Map.js

```diff
@@ -0,0 +1,43 @@
+import GoogleMapReact from 'google-map-react';
+const MapItem = ({ text }) => (
+    <div style={{
+        color: 'white',
+        background: 'purple',
+        padding: '15px 10px',
+        display: 'inline-flex',
+        textAlign: 'center',
+        alignItems: 'center',
+        justifyContent: 'center',
+        borderRadius: '100%',
+        transform: 'translate(-50%, -50%)'
+    }}>
+        {text}
+    </div>
+);
+
+class Map extends React.Component {
```

Comment:
You can change this to:

```suggestion
const Map = ({center = {...}, zoom = 11}) => {
```

and then remove the render() function and just return the jsx and then you've got a function component. You could also set the default props as state if we're expecting them to change: https://reactjs.org/docs/hooks-state.html

Reply:
Didn't realise this would make a suggestion like that but you'd have to put the lat and long in as well.